addons/binding/org.openhab.binding.minecraft/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="minecraft" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="server">
+		<label>Minecraft Bukkit Server</label>
+		<description>Server from which data is being fetched.</description>
+
+		<channels>
+			<channel id="name" typeId="name" />
```

Comment:
Removed both version and also name of server. Don't really know if name is updatable.